drata/manifest.json

```
@@ -0,0 +1,68 @@
+{
+  "display_name": "Drata",
+  "maintainer": "support@drata.com",
+  "manifest_version": "2.0.0",
+  "name": "drata",
+  "app_id": "drata-integration",
+  "app_uuid": "c06736af-282f-4b3c-a9e6-2b049dbc0e2a",
+  "creates_events": false,
+  "short_description": "",
+  "support": "README.md#Support",
+  "public_title": "Drata",
+  "categories": [
+    ""
+  ],
+  "display_on_public_website": true,
+  "legal_terms" : {},
+  "integration_id": "drata",
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Ingest Datadog compliance information to Drata",
+    "title": "Drata",
+    "media": [
+      {
+        "image_url": "images/Drata-001.jpg",
+        "caption": "Put Security & Compliance on Autopilot",
+        "media_type": "image"
+      },
+      {
+        "image_url": "images/Drata-002.jpg",
+        "caption": "Drata automates hundreds of hours of manual, repetitive work to get and stay audit-ready.",
+        "media_type": "image"
+      },
+      {
+        "image_url": "images/Drata-003.jpg",
+        "caption": "Dashboards, Control Mapping across Frameworks, Auditor-Approved Security Policies",
+        "media_type": "image"
+      },
+      {
+        "image_url": "images/Drata-004.jpg",
+        "caption": "Automated Compliance Monitoring of Personnel, Vendor Management Directory, Risk Assessments",
+        "media_type": "image"
+      },
+      {
+        "image_url": "images/Drata-005.jpg",
+        "caption": "Security Control Reporting, Auditor Access Controls",
+        "media_type": "image"
+      }
+    ],
+    "classifier_tags": [
+      "Offering::Integration",
+      "Category::Security",
+      "Category::Compliance",
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows"
+    ]
+  },
+  "author": {
+    "homepage": "https://www.drata.com/",
+    "name": "Drata",
+    "support_email": "support@drata.com",
+    "sales_email": "sales@drata.com",
```

```suggestion
    "sales_email": "sales@drata.com"
```

Should fix the validation job